week-6/Homework/mandatory/1-alarmclock/style.css

```diff
@@ -1,15 +1,85 @@
+body{
+  display: flex;
+  font-family: 'Rye', cursive;
+  font-size: xx-large;
+  background-color:#f06588 ;
+}
 .centre {
   position: fixed;
+  text-align: center;
   top: 50%;
   left: 50%;
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+.settime_font{
+  display: flex;
+  justify-content: space-evenly;
+}
 
 #alarmSet {
-  margin: 20px;
+  margin-bottom:5%;
+  margin-right: 5%;
 }
 
-h1 {
-  text-align: center;
+#timeRemaining{
+  margin-bottom: 15%;
+  font-family: 'Rye', cursive;
+  font-size: 200%;
+}
+#buttons{
+  margin-top: 10%;
 }
+@media(max-width:800px){
+  #timeRemaining{
+    font-size:x-large;
```

Comment:
Try and avoid these terms like `x-large`, it's unclear how big that is. 
Stick to `px` or `rem` values, they are more predictable.